### PR TITLE
8261670: Add javadoc for the XML processing limits

### PR DESCRIPTION
**Pull request by JoeWang-Java**
Add the documentation for XML processing limits to module summary. The limits were previously documented in Java tutorial and guide.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8261670](https://bugs.openjdk.java.net/browse/JDK-8261670): Add javadoc for the XML processing limits

### Reviewers
 * [Lance Andersen](https://openjdk.java.net/census#lancea) (@LanceAndersen - **Reviewer**)
 * [Naoto Sato](https://openjdk.java.net/census#naoto) (@naotoj - **Reviewer**)
 * [Iris Clark](https://openjdk.java.net/census#iris) (@irisclark - **Reviewer**)

### Download
`$ git fetch https://git.openjdk.java.net/jdk pull/2732/head:pull/2732`
`$ git checkout pull/2732`

### CODE REVIEW

File: src/java.xml/share/classes/module-info.java [hunk 1]
```diff
@@ -221,6 +220,85 @@
...
  * <tr>
+ * <th scope="row" style="font-weight:normal" id="EELimit">entityExpansionLimit</th>
+ * <td>Limits the number of entity expansions.
+ * </td>
+ * <td rowspan="9">yes</td>
+ * <td rowspan="9">yes</td>
+ * <td rowspan="9">Integer</td>
+ * <th id="Value" scope="row" style="font-weight:normal" rowspan="9">
+ * A positive integer. A value less than or equal to 0 indicates no limit.
+ * If the value is not an integer, a NumericFormatException is thrown.
```
Comment (naotoj) [inline src/java.xml/share/classes/module-info.java:231]: Is it a ```NumberFormatException```? Also, @link might be useful here.

File: src/java.xml/share/classes/module-info.java [hunk 2]
```diff
@@ -221,6 +220,85 @@
...
+ * </th>
+ * <th id="Default" scope="row" style="font-weight:normal">64000</th>
```
Comment (naotoj) [inline src/java.xml/share/classes/module-info.java:233]: There are other ```Default``` ids in other rows. Should they be distinct? Or rather needed?

File: src/java.xml/share/classes/module-info.java (hunk 1 above)
Comment (JoeWang-Java) [reply src/java.xml/share/classes/module-info.java:231]: Oh my, the typo has been in the tutorial all this time! It's NumberFormatException indeed.

File: src/java.xml/share/classes/module-info.java (hunk 2 above)
Comment (JoeWang-Java) [reply src/java.xml/share/classes/module-info.java:233]: True, not needed. Will remove, and also look at other copy&paste-d ids.

**Review by LanceAndersen — APPROVED** [review]
Looks good Joe.  Added myself as a reviewer to the CSR as well

**Review by naotoj — APPROVED** [review]
Looks good. Thanks for making the changes.

**Review by irisclark — APPROVED** [review]
Looks good.  I'm glad to see useful information from the outdated guides/tutorial being integrated into this documentation.

**Review by LanceAndersen — APPROVED** [review]

**Review by irisclark — APPROVED** [review]

**Review by naotoj — APPROVED** [review]

**Comment by JoeWang-Java** [comment]
Thanks all for the review.
/integrate

**Pull request closed without merging**